src/js/components/AppListComponent.jsx

```diff
@@ -433,16 +433,31 @@ var AppListComponent = React.createClass({
   },
 
   getInlineDialog: function (appNodes = []) {
-    var state = this.state;
+    var {props, state} = this;
```

Comment:
Then let's use different variable names as I otherwise don't see any benefit in "caching" the state var instead of directly accessing it.